scalingo.json

```
@@ -22,13 +22,20 @@
     "first-deploy": "./scripts/scalingo-post-ra-creation.sh"
   },
   "addons": [
-    "postgresql:postgresql-sandbox",
-    "redis:redis-sandbox"
+    {
+      "plan": "postgresql:postgresql-sandbox",
+      "options": {
+        "version": "14"
+      }
+    },
+    {
+      "plan": "redis:redis-sandbox"
+    }
```

Garder le fonctionnement habituel en ne spécifiant pas la version.
```suggestion
    {
      "plan": "postgresql:postgresql-sandbox",
    },
    {
      "plan": "redis:redis-sandbox"
    }
```

---

Corrigé !